lte/cloud/go/services/subscriberdb/servicers/subscriberdb_servicer_test.go

```diff
@@ -41,7 +41,7 @@ func TestSubscriberdbCloudServicer(t *testing.T) {
 	lte_test_init.StartTestService(t)
 	configurator_test_init.StartTestService(t)
 
-	servicer := servicers.NewSubscriberdbServicer()
+	servicer := servicers.NewSubscriberdbServicer(false)
```

Comment:
Nit: passing unadorned booleans is considered an anti-pattern, especially to `new` functions. Consider passing e.g. `SubscriberdbServicerConfig` struct whose only field is that feature flag

Actually, can probably just pass the `Config` object directly that you defined to hold the feature flags. Abstractly it's a bit incongruent but I think it works. Either way, up to you